src/Main.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
+import data from './data.json'
```

Comment:
this is where naming conventions can keep things straight.
When you import the json file, import it as beasts. Ten save it to state as beasts. Than when mapping,
```js
this.state.beasts.map(beast =>
```